### PR TITLE
Make mocked registry more visible

### PR DESCRIPTION
**Pull request by renejeglinsky**
I think this is an important part that could be missed easily. I gave it a try to include it early on.

### CODE REVIEW

**Comment by chgeo** [comment]
As discussed, we should only mention the registry in a 'Deployment/Publishing' section, but not make users install modules from beginning on.

**Comment by danjoa** [comment]
Yes, maybe we need to clarify it's usage: 
- during development / experiments you should work with monorepo setup → `node_modules` in root only
- local installs are required for deployment only, or for simulating isolated development
Important is: never mix both!

**Comment by renejeglinsky** [comment]
I made changes that hopefully match your suggestions. Pls give it a review. Thanks.

File: README.md [hunk 1]
```diff
@@ -53,23 +53,24 @@ npx jest
 ```
 > While mocha is a bit smaller and faster, jest runs tests in parallel and isolation, which allows to run all tests.
 
-
-### Serve `npm`
+### Deploy Sample
 
 We've included a simple npm registry mock, which allows you to do an `npm install @capire/<package>` locally. Use it as follows:
```
Comment (chgeo) [inline README.md:58]: Maybe mention the 'publish and deploy' scenario explicitly: 
```suggestion
To allow publish and deploy tools to execute `npm install` in the individual subfolders, we've included a simple npm registry mock. Use it as follows:
```

**Pull request closed without merging**